### PR TITLE
docs(setup): Update setup readme docs and setup.sh script

### PR DESCRIPTION
**Pull request by damassi**
The type of this PR is: **Docs**

### Description

Updated our README and setup script with the latest.

### CODE REVIEW

File: scripts/setup.sh [hunk 1]
```diff
@@ -13,6 +13,15 @@ else
   echo "Skipping language dependencies installation (asdf not found)"
 fi
 
+if command -v code >/dev/null; then
+  echo "Installing Artsy VSCode extensions..."
+  code --install-extension Artsy.artsy-studio-extension-pack
+  # TODO: Move this into artsy-studio-extension-pack
```
Comment (damassi) [inline scripts/setup.sh:19]: I tried, but I can't figure out for the life of me how to authenticate properly, have tried and tried. I know @MounirDhahri was there recently, so looking for any help 🙏

**Review by anandaroop — APPROVED** [review]
👍🏽

File: scripts/setup.sh (hunk 1 above)
Comment (MounirDhahri) [reply scripts/setup.sh:19]: Sorry for the late reply here. I opened a PR here https://github.com/artsy/vscode-artsy/pull/13

Comment (MounirDhahri) [reply scripts/setup.sh:19]: Authentication here was pretty weird that's why I specified the email last time

**Pull request merged**